eng/scripts/Copy-ApiVersion-Functions.ps1

```diff
@@ -17,8 +16,16 @@ These settings apply only when ``--tag=package-$tagVersion`` is specified on the
 input-file:
 "@
 
-    foreach ($specFile in $specFiles) {
-        $content += "`n  - $baseDir/$specFile"
+    
+    foreach ($resourceProvider in $resourceProviders)
+    {
+        $newDirectory = Join-Path $repoDirectory "specification/$ServiceDirectory/$ServiceType/$resourceProvider/$newApiVersionStatus/$newApiVersion"
```

Comment:
Given how we're importing these functions, `$ServiceDirectory` and other variables *should* be inherited, yes. Did you check that the script actually works as desired? The PR description doesn't say much.